slow_query_test.go

```diff
@@ -0,0 +1,37 @@
+package main
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestParsingDerivedFromValue(t *testing.T) {
```

Comment:
Let's get an example without a partition to show how that impacts things:

```
select * from things where things.id = 1; 
```

and an aliased table 

```
select * from things t where t.id = 1; 
```

and maybe

```
select * from things where id = 1;
``` 

I think those are all valid sql queries that don't reference partitions. I made them up here in the comment section, but ensuring we test the non-success cases is important.

Reply:
dang it, I had tests for when there was not partition/shard. Didn't make it in. Will update